src/components/header/Header.css

```diff
@@ -0,0 +1,30 @@
+@import url("https://fonts.googleapis.com/css2?family=Nothing+You+Could+Do&display=swap");
+
+.header {
+  /* width: 100%;
+    position: fixed;
+    padding: 10px 20px;
+    justify-content: center;
+    background-color: #fff;
+    align-items: center;
+    box-shadow: 0px 0px 15px #dedede; */
```

Comment:
Again, please remove unused and commented out code!

Reply:
ich habe entfernt